packages/fluentui/docs/src/examples/components/Menu/Usage/MenuExampleWithTooltip.shorthand.tsx

```diff
@@ -5,7 +5,7 @@ const itemRenderer = (MenuItem, props) => {
   const { tooltip = '', ...rest } = props;
```

Comment:
we can destructure `key` from here and avoid passing it to `MenuItem` since it's on the `Tooltip` we actually need it.

Reply:
Done